docs/walk-through/parameters.md

```diff
@@ -89,3 +89,34 @@ spec:
 ```
 
 In this workflow, both steps `A` and `B` would have the same log-level set to `INFO` and can easily be changed between workflow submissions using the `-p` flag.
+
+If using the UI, the `enum` parameter type may also be used to restrict the parameter to certain values.
```

Comment:
```suggestion
In the UI, the `enum` field can be used to restrict a parameter to certain values.
```

- more simple and direct, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
- terminology: it's a "field", not a "parameter type"
- grammar: "a" parameter, not specific to "the"

Reply:
```suggestion
With `WorkflowTemplates`, you [can use](../workflow-templates.md#UI) the `enum` field to restrict a parameter to certain values in the UI.
```

After realizing https://github.com/argoproj/argo-workflows/pull/12578#discussion_r1468235830, I think it might be better to remove the example entirely from here and just link to the existing example in the `WorkflowTemplate` docs as that is advanced usage (the walk through is mostly for beginners to / new users of Workflows).

If you'd like, you can also improve that example as part of this PR (e.g. adding versions, improving the wording, perhaps moving it out of its current section)